ember/app/ui/identities/index/template.hbs

```diff
@@ -7,14 +7,29 @@
 
     {{#if (can "add identity")}}
       <div class="uk-flex-none">
-        <button
-          type="button"
-          class="uk-button uk-button-primary"
-          disabled={{this.exportSearch.isRunning}}
-          {{on "click" (perform this.exportSearch)}}
-        >
-          {{t "page.identities.index.export.button"}}
-        </button>
+        <div class="uk-inline">
+          <button
+            type="button"
+            class="uk-button uk-button-primary"
+          >
+            {{t "page.identities.index.export.button"}}
+            <span uk-icon="triangle-down"></span>
+          </button>
+          <div uk-dropdown="mode: click">
+            <ul class="uk-nav uk-dropdown-nav">
+              <li>
+                <a href="#" {{on "click" (perform this.exportSearch "export")}}>
```

Comment:
If you want a click action without a redirect then its most of the time actually a `button` element you need. You can style a button like a link with `uk-button uk-button-link`.

Reply:
Works, but doesn't look the same....
![2021-04-22_11-29-05_324x168](https://user-images.githubusercontent.com/1833932/115691136-0c16b480-a35e-11eb-886e-a41ba843bf6b.png)

Generally it's important you double check design things in my frontend PRs :see_no_evil:

Reply:
Fair, sorry forgot about that. Design-wise is a link in that case probably the better version.

Reply:
Allright, changed it back.